xcell/mappers/mapper_DELS.py

```diff
@@ -85,7 +85,9 @@ def get_catalog(self):
             catalog (Array)
         """
         if self.cat_data is None:
-            fn = f'{self.map_name}_cat.fits'
+            # This will handle the situation when removing the overlap
+            map_name = self.map_name.split("_")[0] + f"_bin{self.zbin}"
+            fn = f'{map_name}_cat.fits'
```

Comment:
Not sure why you need to do this for DELS. This seems to be cutting it in redshift?

Reply:
Yep, that's how it was before too. `self.map_name` was `DELS_binX`. So here I'm just keeping this structure by removing the part from the removed overlap. 

Same in the others

Reply:
I don't understand. So `split("_")[0]` is meant to remove some part of the map name that denotes that you've cut some area?

Reply:
That is correct. This is appended in `MapperBase` https://github.com/xC-ell/xCell/blob/9e6e4ff1e8c6a0fe2641dff5ae1a12400fa76406/xcell/mappers/mapper_base.py#L39-L43

Reply:
oh, wow, that's very niche! OK then.